### ISSUE TITLE
Package pages take a long time to load when package has a large number of files

### ISSUE THREAD

**Issue by pauby** [chocolatey/chocolatey.org#596]
### What You Are Seeing?

When you visit this page https://chocolatey.org/packages/dbatools/ (note that the error happens with any pages that has a lot of files in the package).

I get this error in Waterfox browser:

![image](https://user-images.githubusercontent.com/12760779/45417598-76c9c480-b67a-11e8-8fdd-adb9da180b85.png)

This error, sometimes, in Chrome (sometimes it loads the page but takes a long time):

![image](https://user-images.githubusercontent.com/12760779/45418091-8e557d00-b67b-11e8-81db-0af9bbac31f1.png)

This error in Firefox (again the page takes a long time):

![image](https://user-images.githubusercontent.com/12760779/45418142-a5946a80-b67b-11e8-92c7-0c30a1d50a59.png)

And no error in Edge (although again it takes a long time to load).

When I say a long time it takes anywhere from 30s to 60s to load the page, if it loads.

Because the website is showing all of the files in the package, and the package has many of them, the script runs too long for the browser which results in the errors being displayed or a long time to show the page.

All the browsers tested are the latest available.

### What is Expected?

Given the large number of files it's clear that the page is trying to display them all, causing the issue. I'd like to see it loading a set number of the files (lets say 25 for arguments sake) and then a 'more files' option to see the rest and maybe 'page' through at 25 per 'page'.

### How Did You Get This To Happen? (Steps to Reproduce)

Simply browse the page https://chocolatey.org/packages/dbatools

**Comment by FriedrichWeinmann** [comment]
Quite painful indeed. Sound proposed solution. 👍

**Comment by ferventcoder** [comment]
I agree with this idea - I think there is maybe already an issue listed here for this?

**Comment by ferventcoder** [comment]
Well I didn't find one - but I know we've had a discussion on this in the past and I thought there was an issue in here. Let's keep this one open and put labels on it.
